drivers/hmis/app/jobs/hmis/assessment_questions_job.rb

```diff
@@ -0,0 +1,103 @@
+###
```

Comment:
So you asked about testing.. I think an rspec that tests `Hmis::AssessmentQuestionsJob` is enough? 

For setup:
* to build the assessment with the form processor and values: `build(:hmis_wip_custom_assessment, hud_values: ...)`
* to build the form definition: we don't have a good way of doing this right now. I just proposed a new approach in https://github.com/greenriver/hmis-warehouse/pull/3995 where we can add test definitions in the `form_data/test` environment (it gets loaded because of env var `CLIENT=test`). You could add another one that has just the specific HAT questions that are used for CAS readiness. Does that work? (you can pull out the `item`s from https://qa-hmis.openpath.host/admin/forms/70/edit..)